### PR TITLE
Add weekly update 2016-01-11

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

File: locale/en/blog/weekly-updates/weekly-update.2016-01-11.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+author: Minwoo Jung (@jmwsoft)
+date: 2016-01-11T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-01-11
+layout: blog-post.hbs
+---
+
+### Node.js News — January 11th
+Node v5.4.0 (Stable) is released
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-01-11.md:12]: This gives a feel that v5.4.0 got released on Jan 11th to me.

File: locale/en/blog/weekly-updates/weekly-update.2016-01-11.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### Node v5.4.0 (Stable) Releases
+
+This week we have one release: [Node v5.4.0 (Stable)](https://nodejs.org/en/blog/release/v5.4.0/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-01-11.md]: This week we _had_ one release?

**Comment by JungMinu** [comment]
@thefourtheye Thanks, fixed and updated :) PTAL

**Comment by rosskukulinski** [comment]
Thanks @JungMinu. 

NationJS is having a NodeDay on March 11: http://nationjs.com/

**Comment by JungMinu** [comment]
@rosskukulinski Thanks, updated :)

**Pull request merged**